### PR TITLE
[MIG] stock_move_partner_info v9

### PR DESCRIPTION
**Pull request by serpentcs-dev1**
## Stock Move Partner Info

- This module displays the partner of the picking in its stock movements and also allows grouping by a partner in stock movements.

### CODE REVIEW

**Comment by darshan-serpent** [comment]
pending : readme and manifest file

**Comment by serpentcs-dev1** [comment]
@aheficent Could you please check?

File: stock_move_partner_info/models/stock_move.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+from openerp import fields, models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    picking_partner = fields.Many2one(
+        'res.partner', string='Picking Partner', store=True,
+        related='picking_id.partner_id', help='Partner of the picking')
```
Comment (rousseldenis) [inline stock_move_partner_info/models/stock_move.py]: @darshan-serpent It is preferable to set related as readonly=True

**Review by rousseldenis — CHANGES_REQUESTED** [review]

File: stock_move_partner_info/models/stock_move.py (hunk 1 above)
Comment (serpentcs-dev1) [reply stock_move_partner_info/models/stock_move.py]: Right, of course, I missed that, thanks!

File: stock_move_partner_info/__init__.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 Serv. Tec. Avanzados - Pedro M. Baeza (http://www.serviciosbaeza.com)
```
Comment (serpentcs-dev1) [inline stock_move_partner_info/__init__.py]: @pedrobaeza How should I put this to fit it within 79 characters? Could you please suggest?

Comment (pedrobaeza) [reply stock_move_partner_info/__init__.py]: Please switch this to my new company "Tecnativa". You can put:
```
Copyright 2015 Tecnativa - Pedro M. Baeza
```

**Review by LoisRForgeFlow — CHANGES_REQUESTED** [review]
You have a Invalid view definition, check runbot and travis logs please

**Review by LoisRForgeFlow — CHANGES_REQUESTED** [review]
The groupby is not working too...
image

**Review by AaronHForgeFlow — CHANGES_REQUESTED** [review]
I think the problem in the group_by is because the field is related. I think if we add store=True to the field it'd work.

**Review by LoisRForgeFlow — APPROVED** [review]
This seems fine now! :+1:

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM :+1:

**Comment by StefanRijnhart** [comment]
Because of the way this module was migrated from Odoomrp, the attribution is lost from the commits. @pedrobaeza do you have an opinion on this?

**Comment by pedrobaeza** [comment]
Indeed commit attribution should be respected. Please follow the general migration guide (https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-9.0), but replacing the branch by the odoomrp-wip one.

**Comment by serpentcs-dev1** [comment]
#285 This new PR preserves the commit attribution.

**Pull request closed without merging**